assignments/closure.js

```diff
@@ -1,6 +1,22 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function logic (){
+  let babies = 'All babies are illogical.'
```

Comment:
for closure to happen, you need to have 5 things.

1.  You need to have a overarching function
2. The overarching function needs to return a functional scope which has a inner scoped variable
3. The overarching function needs to return a inner function
4. The overarching function needs to be executed
5. The 2 above items need to be assigned to a variable to keep a hold of that functional scope aka the closure part.